pelicun/assessment.py

```diff
@@ -1609,6 +1668,48 @@ def _loss__add_replacement_time(
     distribution=None,
     theta_1=None,
 ):
+    """
+    Adds replacement time information.
+
+    Parameters
+    ----------
+    adf : pandas.DataFrame
+        Dataframe containing loss information.
+    damage_process_approach : str
+        Supported approaches are 'FEMA P-58', 'Hazus Earthquake -
+        Buildings'.
+    conseq_df : pandas.DataFrame
+        Dataframe containing consequence data for different damage
+        states.
+    occupancy_type : str, optional
+        Type of occupancy, used to look up replacement time in the
+        consequence dataframe for Hazus Earthquake approach. Defaults
+        to None.
+    unit : str, optional
+        Unit for the damage value (e.g., 'worker_day', 'loss_ratio').
```

Comment:
This doesn't make sense to me. Aren't these units for the replacement time value? And, `loss_ratio` is not a valid time unit. I suggest using 'day' besides 'worker_day' as an example.

Reply:
Right, thanks. 63974b8f